### DISCUSSION TITLE
Welcome to IndicWav2Vec Discussions!

### DISCUSSION THREAD

**Discussion by RamanHacks** [AI4Bharat/IndicWav2Vec#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

All Pretrained IndicWav2Vec checkpoints are available on our main [Github Page](https://github.com/AI4Bharat/IndicWav2Vec#resources).
For more details on the model, training and deployment, please visit our [workshop page](https://github.com/AI4Bharat/IndicWav2Vec/tree/main/workshop-2022).

**Comment by 8904591612** [comment]
hello , in the repository  you have  used flashlight.lib.sequence, but that  l module  is not available anywhere , searched for the source code  everywhere  it is not  available, pls help
